### DISCUSSION TITLE
Feature Request: Use token in endpoint URL to pass the last fetch date to the API

### DISCUSSION THREAD

**Discussion by jamealg** [engram-design/FeedMe#883] [Ideas]
We're planning to use FeedMe to sync hundreds of items from a remote API and we'd like to avoid re-syncing the entire thing at a frequent interval. We can pass a `date_modified_after` parameter to our API that will fetch only items that were created/modified after a certain date. It would be really neat if FeedMe could pass the date of the previous run each time the feed is run.

For example, a feed could be configured with the following URL:
```http://myfeed.com/api.json?date_modified_after={date_last_fetched}```

When it runs the first time, it could be left blank:
```http://myfeed.com/api.json?date_modified_after=```

And the second time it would include the date from the previous time it was run:
```http://myfeed.com/api.json?date_modified_after=2018-07-18```

**Comment by weotch** [comment]
This would also be handy for [Google Calendar feeds](https://developers.google.com/calendar/v3/reference/events/list) and it's `syncToken` or `timeMin` parameter.

**Comment by weotch** [comment]
While not **totally** related to the OP, I was able able to make a dynamic Feed URL by using aliases.  In my `general.php` config file I have:


```php
'aliases' => [
  '@calendarFeed' => 'https://www.googleapis.com/calendar/v3/calendars/CALENDAR_ID/events?'.
    'key=API_KEY&singleEvents=true&orderBy=startTime&timeMin='.date('Y-m-d\TH:i:s\Z'),
],
```

And then in FeedMe, just:

![image](https://user-images.githubusercontent.com/77567/117513882-842ccf00-af47-11eb-8549-62e12b3af7a7.png)
